data/result-exporter/core-cne-exporter/src/main/java/com/powsybl/openrao/data/corecneexporter/CoreCneExporter.java

```diff
@@ -33,22 +35,28 @@
 import java.io.StringReader;
 import java.io.StringWriter;
 import java.util.Objects;
+import java.util.Properties;
 
 import static com.powsybl.openrao.data.cneexportercommons.CneConstants.*;
+import static com.powsybl.openrao.data.cneexportercommons.CneUtil.getParametersFromProperties;
+import static com.powsybl.openrao.data.cneexportercommons.CneUtil.getRaoParametersFromProperties;
 
 /**
  * Xml export of the CNE file
  *
  * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
  * @author Peter Mitri {@literal <peter.mitri at rte-france.com>}
  */
-public class CoreCneExporter {
+@AutoService(Exporter.class)
+public class CoreCneExporter implements Exporter {
+    @Override
+    public String getFormat() {
+        return "CORE CNE";
+    }
 
-    public void exportCne(Crac crac,
-                          UcteCracCreationContext cracCreationContext,
-                          RaoResult raoResult, RaoParameters raoParameters,
-                          CneExporterParameters exporterParameters, OutputStream outputStream) {
-        CoreCne cne = new CoreCne(crac, cracCreationContext, raoResult, raoParameters, exporterParameters);
+    @Override
+    public void exportData(RaoResult raoResult, CracCreationContext cracCreationContext, Properties properties, OutputStream outputStream) {
+        CoreCne cne = new CoreCne(cracCreationContext.getCrac(), (UcteCracCreationContext) cracCreationContext, raoResult, getRaoParametersFromProperties(properties), getParametersFromProperties(properties));
```

Comment:
simplify the signature:
```suggestion
        CoreCne cne = new CoreCne((UcteCracCreationContext) cracCreationContext, raoResult, getRaoParametersFromProperties(properties), getParametersFromProperties(properties));
```

also, check the type of context before casting (and throw exception if not ucte)